setup.cfg

```diff
@@ -1,6 +1,6 @@
 [metadata]
 name = visionmetrics
-version = 0.0.10
+version = 0.0.11
```

Comment:
looks like 0.0.10 is not released, no need to bump the version

Reply:
Reverted